source/Hardpoint.h

```diff
@@ -19,6 +19,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Angle.h"
 #include "Point.h"
 
+#include <list>
 #include <vector>
```

Comment:
This can be removed now because all `vectors` are `lists` with these changes, correct?

Reply:
`Hardpoint::Fire` still takes a `vector` of `Projectile`s